torchao/quantization/utils.py

```diff
@@ -357,7 +357,7 @@ def groupwise_affine_quantize_tensor_from_qparams(
     quant_max = 2 ** n_bit - 1
 
     int_data = quantize_affine(w, block_size, scales, zeros, output_dtype, quant_min, quant_max, zero_point_domain = ZeroPointDomain.FLOAT)
-    if TORCH_VERSION_AT_LEAST_2_5:
+    if TORCH_VERSION_AT_LEAST_2_5 and w.shape[-1] > 1:
```

Comment:
will GPTQ be useful for other types of quantization using `quantize_affine`, e.g. fp8? any ideas to generalize the single column stuff to all variations of `quantize_affine`?